lib/config/presets/internal/monorepo.ts

```diff
@@ -215,6 +215,10 @@ const repoGroups = {
   remark: 'https://github.com/remarkjs/remark',
   remix: 'https://github.com/remix-run/remix',
   rjsf: 'https://github.com/rjsf-team/react-jsonschema-form',
+  rollup: [
+    'https://github.com/rollup/rollup',
+    'https://github.com/rollup/plugins',
```

Comment:
should the plugins really be merged together?

Reply:
You might be right.  The different scope and versions is perhaps not that connected as I first thought.

I’ll close this for now and come back to it later if it’s still applicable.